docker-compose.override.yaml

```diff
@@ -8,8 +8,8 @@ services:
     image: fcp-fd-file-processor-development
     container_name: fcp-fd-file-processor-development
     ports:
-      - "3000:3000"
-      - "9229:9229"
+      - "3019:3000"
+      - "9019:9229"
```

Comment:
Can only see the ports being updated in one file. Worth doing a find and replace as the ports will also need updated in the Dockerfile and in the `values.yaml` in the Helm chart. Also instead of `3019:3000` it should be written as `3019:3019` instead of `3019:3000` e.g. you currently have this:
```
    ports:
      - "3019:3000"
      - "9019:9229"
```
but throughout all of our services it's typically written like this:
```
    ports:
      - "3019:3019"
      - "9019:9229"
```